sprocs/assessment_instances_update.sql

```diff
@@ -102,24 +107,24 @@ BEGIN
     -- did we add any instance questions above?
     updated := updated OR (cardinality(new_instance_question_ids) > 0);
 
+    SELECT sum(zmp.max_points)
+    INTO zone_max_points
```

Comment:
Can we name this `zones_total_max_points` or the like (bikeshed away) to indicate that this refers to a sum across all zones and not a single zone? This'll make the later calculation (`GREATEST(zone_max_points - assessment_max_bonus_points, 0)`) a bit clearer.